### PR TITLE
Upgrade raiden-contracts to 0.18.0 and

### PR DESCRIPTION
**Pull request by pirapira**
CONTRACTS_VERSION to 0.11.1 so that the deployments on Görli chain
is accessible.

### CODE REVIEW

**Comment by pirapira** [comment]
If anybody is interested in implementing the rest, please take over.

**Comment by LefterisJP** [comment]
@pirapira I will do it. Let me assign myself in this PR so as to not forget.

**Comment by pirapira** [comment]
Thank you!

**Comment by LefterisJP** [comment]
@pirapira my pleasure. One question. Unit tests fail on startup of production version tests in Goerli. You have not deployed the production version there, only the deployment?

**Comment by pirapira** [comment]
The test failure seems relevant.

**Comment by pirapira** [comment]
@LefterisJP no, I didn't deploy 0.4.0 on Goerli.  Do you want it there?

File: raiden/tests/unit/test_startup.py [hunk 1]
```diff
@@ -135,6 +135,22 @@ def test_setup_contracts():
     assert raiden_contracts_in_data(contracts)
     assert service_contracts_in_data(contracts)
 
+    # Goerli production TODO: Uncomment when production contracts are deployed in Goerli
```
Comment (rakanalh) [inline raiden/tests/unit/test_startup.py]: Can we create a follow up issue for this?

Comment (LefterisJP) [reply raiden/tests/unit/test_startup.py]: done: https://github.com/raiden-network/raiden/issues/3819

**Pull request merged**